reduct/boolean-reduct/rte-helpers.metta

```diff
@@ -14,17 +14,21 @@
 ;;                Only AND nodes have guard sets.
 ;; Example: (AND (NOT A) (NOT B) A) -> ( (NOT A) (NOT B) A)
 (= (getGuardSet $exp)
-    (if (== (car-atom $exp) OR)
-        ()
-        (getLiterals $exp)))
+  (if (isSymbol $exp) (filterLiterals $exp)
+    (if (== $exp ()) ; added is empty checks
+      ()
+      (if (== (car-atom $exp) OR)
+          ()
+          (getLiterals $exp)))))
 
 ;; A function similar to the getGuardSet function but returns a
 ;;    non deterministic list instead of a collapsed atom.
 ;; Example: (AND (NOT A) (NOT B) A) -> [(NOT A), (NOT B), A]
 (= (getGuardSetND $exp)
+  (if (isSymbol $exp) (filterLiterals $exp)
```

Comment:
Same comment here. If the `$exp` is symbol there's no need to give it to the `filterLiterals` function.